compiler/rustc_interface/src/queries.rs

```diff
@@ -107,3 +107,25 @@ impl Linker {
         codegen_backend.link(sess, codegen_results, self.metadata, &self.output_filenames)
     }
 }
+
+/// Checks if the given defid refers to an item that is unnamable, such as one defined in a const block.
+// defined here so it can't accidentally be used directly instead of through the query.
```

Comment:
Not sure to understand what this comment is about. Also not a good idea to put it right after a doc comment. If you want to add code comment after a doc comment, please add an empty line before it.

Reply:
Ah then please remove. If you need to leave comments for reviewers, better use github for that and comment on the code. I tend to do that sometimes as well. Always appreciated when you review code to see the one opening the PR to leave comments for things that need extra attention from reviewer(s).